CMakeLists.txt

```diff
@@ -1,5 +1,19 @@
 cmake_minimum_required(VERSION 3.8)
 
+project(stk-benchmark)
+
+set(BENCHMARK_ENABLE_GTEST_TESTS OFF CACHE BOOL "Do not build benchmark tests")
+if(NOT WIN32)
+    find_package(benchmark)
+endif()
+if(NOT ${benchmark_FOUND})
```

Comment:
This breaks the build on windows (`${benchmark_FOUND}` always evaluates to `true`). For `deform-benchmark` I changed this to `if(NOT benchmark_FOUND)` but I'm not fully sure if this is correct. Does this work for you?

Reply:
Yeah, sorry, that was a typo of mine. The variable must not be referenced in the condition, otherwise it gets a different semantics.